app/src/components/Admin/UserInfo/UserGenericModalComponent.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+import React, { useState } from 'react';
+import AuthServices from '../../../services/auth.services';
+import AdminServices from '../../../services/admin.services';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import GenericModalComponent from '../../GenericModalComponent';
+/*
+This code bassicly extends the GenericModalCompoent
+Where we re use the GenericMopdalComponent, but with the help of the, we add
+some additional fields to the modal, like the name and email of the user.
+
+!!!! Please DONT try to merge the UserGenericModalComponent with the GenericModalComponent!!!!!
+
+*/
```

Comment:
This makes no sense. Generic literally means that it is context insensitive and should be usable everywhere which we should always try to aim for, thats the whole point of a component based framework. There is also nothing about a Modal that is context dependent, it's just a container for content.